From 8fccb754a80969264abb82def5b7fb1a6603c28a Mon Sep 17 00:00:00 2001
From: Lu Hui <luhux76@gmail.com>
Date: Thu, 20 Jul 2023 21:24:46 +0800
Subject: [PATCH 1/2] 
 arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts: change dr_mode
 to peripheral

---
 arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts b/arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts
index 43896723a..fa2060ffb 100644
--- a/arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts
+++ b/arch/arm/boot/dts/allwinner/suniv-f1c100s-licheepi-nano.dts
@@ -63,7 +63,7 @@ &uart0 {
 };
 
 &usb_otg {
-	dr_mode = "otg";
+	dr_mode = "peripheral";
 	status = "okay";
 };
 
-- 
2.41.0

